holidays/countries/sao_tome_and_principe.py

```
@@ -0,0 +1,121 @@
+        "Caué": "03",
+        "Lembá": "04",
+        "Lobata": "05",
+        "Mé-Zóchi": "06",
+        # Autonomous Region.
+        "Príncipe": "P",  
+    }
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, cls=SaoTomeAndPrincipeStaticHolidays)
```

Ruff (0.8.2)

Undefined name `SaoTomeAndPrincipeStaticHolidays`

(F821)